examples/route_guide/client/client.go

```diff
@@ -63,15 +63,15 @@ func printFeatures(client pb.RouteGuideClient, rect *pb.Rectangle) {
 	defer cancel()
 	stream, err := client.ListFeatures(ctx, rect)
 	if err != nil {
-		log.Fatalf("%v.ListFeatures(_) = _, %v", client, err)
+		log.Fatalf("client.ListFeatures failed: %v", err)
 	}
 	for {
 		feature, err := stream.Recv()
 		if err == io.EOF {
 			break
 		}
 		if err != nil {
-			log.Fatalf("%v.ListFeatures(_) = _, %v", client, err)
+			log.Fatalf("stream.Recv failed: %v", err)
```

Comment:
Please make this `client.ListFeatures failed: %v` like above.  This is the status for the whole RPC if it errors, not just the `Recv` operation.